documentation/modules/configuring/con-config-kafka-zookeeper.adoc

```diff
@@ -80,8 +91,10 @@ spec:
             secretName: my-secret
             certificate: my-certificate.crt
             key: my-key.key
+    # Authorization (optional)
     authorization:
       type: simple
+    # Kafka configuration (optional)
```

Comment:
Maybe this could be recommended as some of these options default to values such as `1` and do not provide reliability / availability out of the box?